pages/Basic-Machines.md

```diff
@@ -0,0 +1,24 @@
+# Basic Machines
+The Basic Machines category includes various machines that have simple functionalities and do not require any electricity.
+
+## Items
```

Comment:
Should rather be "Table of Contents" or at least "Machines", since they are not Items.